src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -539,19 +539,27 @@ function updateNowPlayingInfo(state) {
             if (textLines[1].text) {
                 const text = document.createElement('a');
                 text.innerText = textLines[1].text;
+                documentTitle = text.innerText;
                 secondaryText.appendChild(text);
             }
         }
 
         if (textLines[0].text) {
             const text = document.createElement('a');
             text.innerText = textLines[0].text;
+            if (documentTitle != 'Jellyfin') {
+                documentTitle += ' - ';
+            }
+            documentTitle += text.innerText;
             itemText.appendChild(text);
         }
+
         nowPlayingTextElement.appendChild(itemText);
         nowPlayingTextElement.appendChild(secondaryText);
     }
 
+    document.title = documentTitle;
```

Comment:
```suggestion
    document.title = documentTitle || 'Jellyfin';
```

We can set `documentTitle` to initially be an empty string for improved readability.